terraform/maas/rds.tf

```diff
@@ -0,0 +1,66 @@
+resource "aws_db_subnet_group" "maas_db_subnet_group" {
+  name        = "maas-dbgrp"
+  description = "Maas DB subnet group"
+  subnet_ids  = data.aws_subnet_ids.public_subnets.ids
+
+  tags = merge(
+    local.common_tags,
+    map(
+      "Name", "maas-dbgrp"
+    )
+  )
+}
+
+resource "aws_security_group" "maas_postgres_sg" {
+  name        = "maas-postgres-sg"
+  description = "Maas postgres security group"
+  vpc_id      = join(", ", data.aws_vpcs.moz_internal_us_west_2.ids)
+
+  ingress {
+    from_port       = 5432
+    to_port         = 5432
+    protocol        = "tcp"
+    security_groups = [aws_security_group.maas_ec2_sg.id]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = merge(
+    local.common_tags,
+    map(
+      "Name", "maas-postgres-sg"
+    )
+  )
+}
+
+
+resource "aws_db_instance" "maas_postgres" {
+  identifier                 = "maas-postgres"
+  storage_type               = "gp2"
+  allocated_storage          = 20
```

Comment:
20gb is ok for a start but we will need to up this as we import images

Reply:
is this limited or will the storage grow to fit usage without changing this?

Reply:
It will not scale automatically since we want to prevent run away storage costs.  I totally expect this to be manually increased when we can determine the size needed.  We should also monitor for running out of storage and react accordingly.